samples/metrics/tracing/sample.md

```diff
@@ -1,7 +1,8 @@
 ---
 title: Sending metrics data to trace log
 component: Metrics
-reviewed: 2017-04-04
+versions: '[1.0,1.1)'
```

Comment:
What about a case when we introduce a new API that we'd like encourage people to use? Shouldn't I use versions for it?